### ISSUE TITLE
Move Ceph cluster from PRB to MGHPCC and prmote it into new research ceph cluster 

### ISSUE THREAD

**Issue by msdisme** [CCI-MOC/ops-issues#991]
@pjd-nu  and @naved001 discussed over lunch on Jun 16 - Peter this ticket is to capture your thoughts and discuss what support or space may be needed.

**Comment by joachimweyl** [comment]
Current plan:
- [ ] Get Flax to bring back some hardware from the 2 Sigma donation
   - 3 FX2s (2 FC630s & 2 FD332s in each) (6 each all together)
      -  8 SSD & 8 HDD in each FD332 (16 each in each FX2) (48 each all together)
   - 1 FX2 (4 FC630s)
- [ ] Gather Hardware from old PRB cluster
   - 32 SSD
   - 48 HDD
- [ ] Order extra hardware
   - 16 SSDs
   - all required Sleds (96? do we have any sleds?)
- [ ] Setup Ceph cluster
This is a lot of steps, should this be broken down into multiple issues?

**Comment by joachimweyl** [comment]
@pjd-nu is this still the desired plan? we will need to let Flax know soon before they finalize the processing of these units.

**Comment by joachimweyl** [comment]
@er1p agreed [here](https://massopencloud.slack.com/archives/C051SVA0PF0/p1692366881761379) to set aside:
- 5 FX2s
- 10 FC630s
- 8 FD332s

**Comment by pjd-nu** [comment]
Just checking on how this is doing, as we'll have to coordinate taking down the PRB cluster and swapping some cards and disks in order to set these up. (note FC630 and FC830 are exchangeable here - if there are differences in the blades, using the lower-performance ones here would make sense)

**Comment by joachimweyl** [comment]
Harlan McCanne from Flax confirms they have reserved the hardware and will bring it to MGHPCC when we are ready to start work on this project.

**Comment by joachimweyl** [comment]
@pjd-nu the hardware for this is at Flax. We can ask them to bring it back to MGHPCC when we are ready for it to be used. They wanted to be clear that the FD332s are currently diskless. I believe you already knew this based on your suggestion of gathering disks but it is better safe than sorry.

**Comment by pjd-nu** [comment]
Still waiting for the eval hardware. Any update???

**Comment by msdisme** [comment]
Drives @er1p held onto at flax: https://github.com/CCI-MOC/ops-issues/issues/1038#issuecomment-1793843794

**Comment by joachimweyl** [comment]
we should be able to get all the drives we need from [this](https://github.com/CCI-MOC/ops-issues/issues/1038#issuecomment-1793843794) batch.

**Comment by joachimweyl** [comment]
Erik already delivered a test model of one of the FX2s to @pjd-nu 
1 FX2
3 FC630
 - 2 1.6TB SAS SSD
1 FD332
 - 16 1.6TB SAS SSD

**Comment by joachimweyl** [comment]
This is the hardware we still need:
3 FX2
7 FC630
- 2 SAS SSDs in front bays each
5 FD332
- 32 1.6TB SAS drives (loose or in FD332)

**Comment by msdisme** [comment]
Row 4, Pad A, Cabinet 4 is being.

**Comment by joachimweyl** [comment]
@er1p can we have the rest of the parts for this cluster sent to the MGHPCC?

**Comment by joachimweyl** [comment]
@er1p 
"those servers were delivered yesterday
FC630 / storage servers in R4-PA-C04

![image](https://github.com/CCI-MOC/ops-issues/assets/28782106/50089b6d-2856-4378-b180-42720f67cd0b)
![image](https://github.com/CCI-MOC/ops-issues/assets/28782106/47b565c4-b3d1-4d48-be34-66f9b48207cf)

we over-delivered a small amount
four chassis FX2 - request was three
eight servers FC630 - request was seven
six JBODs FD332 - request was five
fifty-six drives 1.6TB SAS/SSD - request was forty-six
when we were putting the nodes together, moving around the JBOD units seemed to move the PCI structure around, and seemed to get confusing very quickly, so we figured it made more sense to be consistent - specifically, we created ::
compute/FC630 node
hosting for up to four FC630 nodes, no JBOD
1 or 2 or 3 or 4 servers - one or two drives each
storage/FC630 node
hosting for up to two FC630 nodes in the top row
hosting for up to two FD332 JBODs in the bottom row
1 or 2 servers - one or two drives each
1 or 2 JBODs - up to 16x drives each
so then it was logical to deliver
four chassis
one compute - two servers FC630
three storage - six servers FC630, six JBOD FD332
56x drives total - sixteen drives (2 per server) and forty drives (16 + 16 + 2 + 2 + 2 +2 in jbod)
this way every jbod has at least two drives, so fully tested / working
this means that Peter can move the drives around at will - and it means that if he wants to merge the systems that he has on campus, he can just move the servers and loose drives around directly into the rack, he doesn't have to mount another chassis"

**Comment by joachimweyl** [comment]
@pjd-nu ^

**Comment by pjd-nu** [comment]
Thanks!
